src/Domain/Person.php

```diff
@@ -68,6 +64,22 @@ class Person
      */
     public array $cash_balance = [];
 
+    /**
+     * @OA\Property(
+     *     type="object",
+     *     description="Properties are lowercase currency codes, e.g. 'gbp'. Values are
+     *     available amounts in smallest denomination, e.g. 123 pence.",
+     *     example={
+     *         "eur": 0,
+     *         "gbp": 123,
+     *     }
+     * )
+     * @var null|int[]  Pending Payment Intents for Big Give (i.e. donor fund top up tips)
+     *                  in smallest unit (cents/pence), keyed on lowercase currency code.
+     *                  Or may be null if no tip balances requested on Get.
```

Comment:
Feels awkward to have something in the domain depending on what was or wasn't requested on Get - but I can see why it's done that way, this is sort of a combination of a domain model and a DTO.